ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetConfigSummary.java

```diff
@@ -0,0 +1,81 @@
+package com.spectralogic.ds3cli.command;
+
+import com.spectralogic.ds3cli.*;
+import com.spectralogic.ds3cli.exceptions.CommandException;
+import com.spectralogic.ds3cli.models.GetConfigSummaryResult;
+import com.spectralogic.ds3cli.models.Result;
+
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+public class GetConfigSummary extends CliCommand<GetConfigSummaryResult> {
```

Comment:
It would be helpful to have some documentation describing what this command does.